<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contactedit</class>
 <widget class="QDialog" name="contactedit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>433</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>653</width>
    <height>433</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>433</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ÂàõÂª∫Êñ∞ÁöÑËÅîÁ≥ª‰∫∫</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>581</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ÂßìÂêç</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_edit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ÁîµËØù</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tel_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>EMail</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mail_edit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>QQ/Wechat</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="qq_edit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>ÁîüÊó•</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>ÊÄßÂà´</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gender_combo">
        <item>
         <property name="text">
          <string>Áî∑</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Â•≥</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ë∑®ÊÄßÂà´</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>ÂàÜÁ±ª</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="sort_combo">
        <item>
         <property name="text">
          <string>Âèå‰∫≤</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÂÖÑÂºüÂßêÂ¶π</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>‰∫≤Êàö</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Áà±‰∫∫</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÊúãÂèã</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Âêå‰∫ã</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ÂêåÂ≠¶</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ËÄÅÂ∏à</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>‰∏äÂè∏</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>‰∏ãÂ±û</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Âú∞ÂùÄ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="add_edit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="note_textedit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>531</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="savet_Button">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>ÂèñÊ∂à</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Â§áÊ≥®</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>*ÂßìÂêç‰∏çÂèØ‰ª•‰øÆÊîπüòØ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>‰øùÂ≠ò</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
